### PR TITLE
feat: support `--vars` option in the CLI

### PR DESCRIPTION
**Pull request by yu-iskw**
Closes: https://github.com/lightdash/lightdash/issues/6098

### Description:
The `--vars` option is supported in some sub commands of Lightdash CLI. However, `CompileHandlerOptions` doesn't have the `vars` field.

### CODE REVIEW

**Comment by rephus** [comment]
Looks good, testing on render

**Comment by rephus** [comment]
There is an error on the e2e test : 

image

**Review by rephus — CHANGES_REQUESTED** [review]
See error

**Comment by yu-iskw** [comment]
@rephus Sure. I will look into it.

File: packages/cli/src/handlers/dbt/run.ts [hunk 1]
```diff
@@ -3,17 +3,13 @@ import execa from 'execa';
...
+import { DbtCompileOptions } from './compile';
 
-type DbtRunHandlerOptions = {
-    profilesDir: string;
-    projectDir: string;
-    target: string | undefined;
-    profile: string | undefined;
-    select: string[] | undefined;
-    models: string[] | undefined;
+type DbtRunHandlerOptions = DbtCompileOptions & {
```
Comment (yu-iskw) [inline packages/cli/src/handlers/dbt/run.ts:8]: Some type aliases like `GenerateHandlerOptions` take advantage of `CompileHandlerOptions`. But, I feel `DbtCompileOptions` would look more natural in this case, as `DbtRunHandlerOptions` isn't exposed as command line options.

**Review by rephus — APPROVED** [review]

**Pull request merged**